example/android/app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar.Bridge">
+    <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
```

Comment:
Shouldn't this be a `MaterialComponents` theme?

Reply:
This is default RN setup. 
This helps Example app to stay as "raw" as possible 😁